src/main/java/com/entropyteam/entropay/employees/jobs/HolidayJob.java

```diff
@@ -0,0 +1,45 @@
+package com.entropyteam.entropay.employees.jobs;
+
+import com.entropyteam.entropay.employees.dtos.CalendarEventDto;
+import com.entropyteam.entropay.employees.repositories.HolidayRepository;
+import com.entropyteam.entropay.employees.services.GoogleService;
+import com.entropyteam.entropay.employees.services.HolidayService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+
+@Component
+public class HolidayJob {
+    private static final Logger LOGGER = LogManager.getLogger();
+    private final HolidayRepository holidayRepository;
+    private final HolidayService holidayService;
+    private final GoogleService googleService;
+
+
+    @Autowired
+    public HolidayJob(HolidayRepository holidayRepository, HolidayService holidayService, GoogleService googleService) {
+        this.holidayRepository = holidayRepository;
+        this.holidayService = holidayService;
+        this.googleService = googleService;
+    }
+
+    //Job to execute in January
+    @Scheduled(cron = "0 0 9 1 1 ?")
```

Comment:
I don't think a `@Scheduled` is needed. probably at that time none of the holidays will be set in the app.
Just remove the annotation and we can keep only the job to call it from the endpoint